### PR TITLE
Update merge pointers to clear up merge collision error

### PR DESCRIPTION
**Pull request by curlup**

### CODE REVIEW

File: cidc_schemas/schemas/templates/metadata/mif_template.json [hunk 1]
```diff
@@ -56,133 +56,132 @@
...
         "data_columns": {
           "ROIs": {
             "CIMAC ID": {
-              "merge_pointer": "5/cimac_id",
+              "merge_pointer": "3/cimac_id",
               "type_ref": "sample.json#properties/cimac_id"
             },
             "ROI id": {
-              "merge_pointer": "2/roi_id",
+              "merge_pointer": "0/exports/-/roi_id",
```
Comment (curlup) [inline cidc_schemas/schemas/templates/metadata/mif_template.json:67]:
```suggestion
              "merge_pointer": "0/roi_id",
```

File: cidc_schemas/schemas/templates/metadata/mif_template.json [hunk 2]
```diff
@@ -56,133 +56,132 @@
...
             "Export ID": {
               "description": "The export as given by the folder name. Generally the marker(s) used in each analysis.",
-              "merge_pointer": "0/export_id",
+              "merge_pointer": "0/exports/-/export_id",
               "type_ref": "assays/components/imaging/mif_export.json#properties/export_id"
             },
             "Score data files": {
               "description": "Comma separated list of score_data.txt files from inform export, will be usually just one.",
-              "merge_pointer": "0/score_data/-",
+              "merge_pointer": "0/exports/-/score_data/-",
```
Comment (curlup) [inline cidc_schemas/schemas/templates/metadata/mif_template.json:77]:
```suggestion
              "merge_pointer": "0/exports/0/score_data/-",
```

File: cidc_schemas/schemas/templates/metadata/mif_template.json [hunk 3]
```diff
@@ -56,133 +56,132 @@
...
+                  "parse_through": "lambda prefix: str(prefix) + 'image_with_phenotype_map.tif'",
+                  "merge_pointer": "0/exports/-/image_with_phenotype_map",
+                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/image_with_phenotype_map.tif",
+                  "type_ref": "assays/components/local_file.json#properties/file_path",
+                  "is_artifact": 1,
+                  "allow_empty": true
+                },
+                {
+                  "parse_through": "lambda prefix: str(prefix) + 'image_with_tissue_seg.tif'",
+                  "merge_pointer": "0/exports/-/image_with_tissue_seg",
```
Comment (curlup) [inline cidc_schemas/schemas/templates/metadata/mif_template.json:179]:
```suggestion
                  "merge_pointer": "0/exports/0/binary_seg_maps",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/binary_seg_maps.tif",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'cell_seg_data.txt'",
                  "description": "Comma separated list of cell_seg_data.txt files from inform export",
                  "merge_pointer": "0/exports/0/cell_seg_data",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/cell_seg_data.txt",
                  "is_artifact": 1
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'cell_seg_data_summary.txt'",
                  "merge_pointer": "0/exports/0/cell_seg_data_summary",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/cell_seg_data_summary.txt",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'phenotype_map.tif'",
                  "merge_pointer": "0/exports/0/phenotype_map",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/phenotype_map.tif",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'tissue_seg_data.txt'",
                  "merge_pointer": "0/exports/0/tissue_seg_data",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/tissue_seg_data.txt",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1,
                  "allow_empty": true
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'tissue_seg_data_summary.txt'",
                  "merge_pointer": "0/exports/0/tissue_seg_data_summary",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/tissue_seg_data_summary.txt",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1,
                  "allow_empty": true
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'image_with_all_seg.tif'",
                  "merge_pointer": "0/exports/0/image_with_all_seg",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/image_with_all_seg.tif",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1,
                  "allow_empty": true
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'image_with_cell_seg_map.tif'",
                  "merge_pointer": "0/exports/0/image_with_cell_seg_map",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/image_with_cell_seg_map.tif",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1,
                  "allow_empty": true
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'image_with_phenotype_map.tif'",
                  "merge_pointer": "0/exports/0/image_with_phenotype_map",
                  "gcs_uri_format": "{protocol identifier}/mif/{cimac id}/roi_{roi id}/{export id}/image_with_phenotype_map.tif",
                  "type_ref": "assays/components/local_file.json#properties/file_path",
                  "is_artifact": 1,
                  "allow_empty": true
                },
                {
                  "parse_through": "lambda prefix: str(prefix) + 'image_with_tissue_seg.tif'",
                  "merge_pointer": "0/exports/0/image_with_tissue_seg",
```

**Review by curlup — COMMENTED** [review]
I think something like that might work

**Comment by scvannost** [comment]
No longer needed

**Pull request closed without merging**